pkg/querier/distributor_queryable_streaming_test.go

```diff
@@ -23,15 +23,15 @@ import (
 	"github.com/grafana/mimir/pkg/util/limiter"
 )
 
-func TestStreamingChunkSeries_HappyPath(t *testing.T) {
-	chunkIteratorFunc := func(_ chunkenc.Iterator, chunks []chunk.Chunk, from, through model.Time) chunkenc.Iterator {
-		return streamingChunkSeriesTestIterator{
-			chunks:  chunks,
-			from:    from,
-			through: through,
-		}
+var streamingChunkSeriesTestIteratorFunc = func(_ chunkenc.Iterator, chunks []chunk.Chunk, from, through model.Time) chunkenc.Iterator {
+	return streamingChunkSeriesTestIterator{
+		chunks:  chunks,
+		from:    from,
+		through: through,
 	}
+}
```

Comment:
can this become a regular function?
```suggestion
func streamingChunkSeriesTestIteratorFunc(_ chunkenc.Iterator, chunks []chunk.Chunk, from, through model.Time) chunkenc.Iterator {
	return streamingChunkSeriesTestIterator{
		chunks:  chunks,
		from:    from,
		through: through,
	}
}
```

Reply:
Yes it can, good catch.